docs/markdown/30-testing/01-others.md

```diff
@@ -13,4 +13,88 @@ If both cases, you can use **bats** to help you write a test suite for your acti
 
 # Testing Shell script with Bats
 
-TODO example
+## Test folder
+
+```yaml
+.github/
+    actions/
+        runs-using-docker/
+            cleanup.sh
+            main.sh
+            ...
+test/
+    bats/               <- submodule
+    test_helper/
+        bats-support/   <- submodule
+        bats-assert/    <- submodule
+    test.bats
+    ...
+
+```
+
+
+##--##
+
+# Testing Shell script with Bats
+
+## **test.bats** 
+
+```
+setup() {
```

Comment:
maybe add a line about how `setup` is run (or just a Speaker note to remember to tell it)

Reply:
Done + added some things about the teardown() method